bootstrap.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
you took it out of unittest.sh but I don't see the diff where the unittest.sh calls this script - forgot to check it in?

Reply:
Fixed